modules/configuration-ovnk-multi-network-policy.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-additional-network.adoc
+
+:_content-type: CONCEPT
+[id="compatibility-with-multi-network-policy_{context}"]
+= Compatibility with multi-network policy
+
+The multi-network policy API, provided by `MultiNetworkPolicy` CRD in the `k8s.cni.cncf.io` API group, is compatible with an OVN-Kubernetes secondary network. When defining a network policy, what network policy rules can be used depends on whether the OVN-Kubernetes secondary network defines the `subnets` field. Refer to the following table for details:
+
+.Supported multi-network policy selectors based on `subnets` CNI configuration
+[cols=".^3,.^7",options="header"]
+|====
+a|`subnets` field specified|Allowed multi-network policy selectors
+
+|
+Yes
+a|
+* `podSelector` and `namespaceSelector`
+* `ipBlock`
+
+|
+No
+a|
+* `ipBlock`
+
+|====
+
+For example, the following multi-network policy is valid only if the `subnets` field is defined in the additional network CNI configuration for the additional network named`blue2`:
```

Comment:
`blue2` doesn't render well. I think it's because there isn't a space character before it.

```suggestion
For example, the following multi-network policy is valid only if the `subnets` field is defined in the additional network CNI configuration for the additional network named `blue2`:
```